dp_wizard/utils/code_generators/abstract_generator.py

```diff
@@ -70,8 +70,7 @@ def template():
             )
             .finish()
         )
-        # Line length determined by PDF rendering.
-        return black.format_str(code, mode=black.Mode(line_length=74))  # type: ignore
```

Comment:
Why is the formatting removed here?

Reply:
I moved black formatting into DP Wizard Templates, but it is only applied to the final notebook render by default, and is optional when just doing a single template... but since we show code snips to users, it would be good to make sure it's still called here, through the library.

Reply:
actually, we only see this output once it has been transformed into a notebook, so black is applied automatically.